src/coreclr/nativeaot/System.Private.CoreLib/src/System/Runtime/InteropServices/ComWrappers.NativeAot.cs

```diff
@@ -497,9 +494,9 @@ internal unsafe class NativeObjectWrapper
             internal IntPtr _externalComObject;
             private IntPtr _inner;
             internal ComWrappers _comWrappers;
-            internal readonly GCHandle _proxyHandle;
             internal readonly GCHandle _proxyHandleTrackingResurrection;
```

Comment:
I do not think it is an improvement to move ProxyHandle into a property in the middle of the implementation. For types like this, it is important to see the whole state in one place.